writing/paper.md

```diff
@@ -2,84 +2,243 @@
+
+Accuracy: 0.9838565022421525
+
+recision: 0.9852941176470589
+
+Recall: 0.8933333333333333
+
+F1 score: 0.9370629370629371
+
+These metrics show that the classifier effectively categorizes both spam and non-spam messages. The classifier's high accuracy, precision, recall, and F1 score values imply that it can tell spam messages apart from non-spam messages.
+It's crucial to remember that the type of dataset used and the characteristics of the messages can have an impact on how well the classifier performs. To evaluate the classifier's effectiveness on various datasets and look into potential enhancements or optimizations, additional experimentation and evaluation are possible.
+
+![Screenshot 2023-07-27 at 1 04 46 PM 2](https://github.com/LS102-summer2023/project-mishikakansal/assets/136472000/0f2a9615-9003-4de4-b777-15ae3530bcd6)     (22)
 
-TODO: Especially as it pertains to responsible computing, if conducting experiments or evaluations that involve particular ethical considerations, detail those issues here.
 
 #### Evaluation
 
-TODO
+The search results offer information on the most recent developments in natural language processing( NLP) and cybersecurity. The following are highlighted by the results:
+
+1. As it can assist with breach protection, identification, and scale and scope analysis, machine learning (ML) is integrating into cybersecurity workflows more and more.
+
+2. NLP, an AI-based deep learning branch that focuses on how people and computers interact using common language, offers a wealth of capabilities to enhance human ability. NLP is used to find threats and overlaps in data, standards, and frameworks in an organization's tech stack.
+
+3. NLP is being used to automate processes like framework crosswalking and make security telemetry actionable from a risk and compliance standpoint.
+
+4. NLP can be used to understand the internal structure of an email, spot spammer patterns, and analyze bot or spam behavior in text.
+
+5. The creation of new NLP-based systems for cybersecurity and digital forensics is one of the ongoing research projects in the field.
+
+High accuracy, precision, recall, and F1 score values were obtained in the prototype Naive Bayes algorithm spam message classifier implementation, demonstrating its ability to effectively classify both spam and non-spam messages. The performance of the classifier, however, can be influenced by the kind of dataset used and the messages' characteristics. It is possible to conduct additional experiments and evaluate the classifier's performance on various datasets in order to identify any potential improvements.
+
+The project's objectives include identifying the most efficient NLP methods for spotting and preventing cyberattacks, incorporating them into current cyber workflows to help reduce breaches and protect users, offering cybersecurity professionals useful insights and real-world applications, and breaking them down into manageable steps.
```

Comment:
It is still not clear how and where these objectives were achieved in this paper.